src/System.Net.Sockets/src/System.Net.Sockets.builds

```diff
@@ -11,6 +11,13 @@
     <Project Include="System.Net.Sockets.csproj">
       <OSGroup>Windows_NT</OSGroup>
     </Project>
+    <Project Include="System.Net.Sockets.csproj">
+      <TargetGroup>net46</TargetGroup>
+    </Project>
+    <Project Include="System.Net.Sockets.csproj">
+      <TargetGroup>netcore50</TargetGroup>
```

Comment:
Nit: OSGroup above TargetGroup. I'm going to run a script on the repo again soon to make these consistent and to help identify any miss matches between .builds and the configurations in the project files.